app/addons/fauxton/components.react.jsx

```diff
@@ -403,7 +405,7 @@ function (app, FauxtonAPI, React, ReactDOM, ZeroClipboard, ReactBootstrap) {
             {content}
           </Modal.Body>
           <Modal.Footer>
-            <button className="btn btn-success js-btn-success" onClick={this.props.onSubmit}>
+            <button className={btnClasses} onClick={this.props.onSubmit}>
               <i className="fonticon-ok-circled"></i> {this.props.successButtonLabel}
             </button>
```

Comment:
Yeah, I guess I can refactor that component to allow passing the custom classes.